modules/roles_profiles/manifests/profiles/gecko_3_t_osx_1014_generic_worker.pp

```diff
@@ -21,9 +21,9 @@
                 telegraf_user       => lookup('telegraf.user'),
                 telegraf_password   => lookup('telegraf.password'),
                 puppet_env          => 'dev',
-                puppet_repo         => 'https://github.com/davehouse/ronin_puppet.git',
-                puppet_branch       => '1561956_generic-worker_15-recover',
-                puppet_notify_email => 'dhouse@mozilla.com',
+                puppet_repo         => 'https://github.com/mozilla-platform-ops/ronin_puppet.git',
+                puppet_branch       => 'master',
```

Comment:
With the builder branch from Dave's repo, there was 100+ commits difference. It maybe worth checking before pinning to master on this repo.

Reply:
If you look at https://github.com/mozilla-platform-ops/ronin_puppet/tree/1561956_generic-worker_15-recover it just says `588 commits behind master` with nothing ahead.

Reply:
Can we test the a Mac tester against the master branch before merging this? That is such a large difference there is maybe something that breaks.

Reply:
True. We can do that or not move to master and leave it as-is.

Reply:
Let's leave it as-is and not make our lives extra hard.